lib/bolt/node/ssh.rb

```diff
@@ -68,12 +78,22 @@ def execute(command, options = {})
           raise "could not execute command: #{command.inspect}" unless success
 
           channel.on_data do |_, data|
-            result_output.stdout << data
+            if use_sudo && data == sudo_prompt
+              channel.send_data "#{@password}\n"
```

Comment:
Should this be `sudo_password`?

Reply:
yes, per TODO list in the pr description